test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -127,7 +144,12 @@ static void test(Path top) throws IOException {
 
     public static void main(String[] args) throws IOException {
         // create temporary directory to run tests
-        Path dir = TestUtil.createTemporaryDirectory();
+        Path dir;
+        if(0 == args.length) {
```

Comment:
Can you change this to `if (args.length == 0)`  ?

Reply:
Thanks for the review, the code has been changed.